src/devices/Pn532/Pn532.cs

```diff
@@ -25,24 +25,37 @@ namespace Iot.Device.Pn532
     /// </summary>
     public class Pn532 : CardTransceiver, IDisposable
     {
+        private const int I2cMaxBuffer = 1024;
+
         // Communication way
         private const byte ToHostCheckSumD5 = 0xD5;
+
         private const byte FromHostCheckSumD4 = 0xD4;
+
         // Preamble, codes and postamble
         private const byte Preamble = 0x00;
         private const byte Postamble = 0x00;
         private const byte StartCode1 = 0x00;
+
         private const byte StartCode2 = 0xFF;
+
         // Operation type for SPI
         private const byte WriteData = 0b0000_0001;
         private const byte ReadStatus = 0b0000_0010;
+
         private const byte ReadData = 0b0000_0011;
+
         // Acknowledge
-        private byte[] AckBuffer = { 0x00, 0x00, 0xFF, 0x00, 0xFF, 0x00 };
+        private byte[] _ackBuffer = { 0x00, 0x00, 0xFF, 0x00, 0xFF, 0x00 };
```

Comment:
What is the logic applied to the above new lines that were added to this file?

Reply:
A comment line should be preceeded by an empty line, but it seems some extra lines got introduced (maybe due to moving members around). I'll fix it.